SimMuon/MCTruth/python/SimMuFilter_cfi.py

```diff
@@ -8,3 +8,4 @@
               simHitsMuonDTInput = cms.InputTag("g4SimHits","MuonDTHits","SIM")
 )
 
+SimMuFiltSeq = cms.Sequence(SimMuFilter)
```

Comment:
hi @kskovpen - you should define this sequence in a cff file instead.  (eg, SimMuFiltSeq_cff.py that will be just a few lines)